Documentation/compatibility/interop-enable-winrt-iagileobject.md

```diff
@@ -0,0 +1,56 @@
+## .NET Interop will now QI for IAgileObject (a WinRT interface)
+
+### Scope
+Edge
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+When using a WinRT event with a .NET delegate, Windows will QI for IAgileObject.  Prior to this change the runtime would fail that QI and the event could not be subscribed.
+
+- [ x ] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+If enabling the QI for IAgileObject is thought to break execution, you can disable this code by setting the following configuration.
+
+#### Method 1: environment variable
+
+Set the following environment variable:
+
+        COMPLUS_DisableCCWSupportIAgileObject=1
```

Comment:
Instead of plain text, this should be fenced. It only needs to be indented 3 spaces.